src/styles/theme-manager.js

```diff
@@ -208,7 +209,7 @@ export default {
         borderColor: rawTheme.palette.borderColor,
       },
       isRtl: false,
-      zIndex: rawTheme.zIndex,
+      zIndex: zIndex,
```

Comment:
The idea of the rawTheme is to build the muiTheme.
So I think that the zIndex has nothing to do inside the rawTheme, as the isRtl.